src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -688,6 +746,16 @@ export default class TaskRoleContainerList extends React.Component {
                           'all',
                         ),
                     },
+                    {
+                      key: 'trackingPage',
+                      name: 'Browse log folder',
```

Comment:
Since `log folder` is no longer available.  I think it should be `Show All Logs` instead of `Browse log folder`.

@Binyang2014  Do you have any comment on this?

Reply:
I'm fine with `Show All Logs`